browser/resources/settings/brave_routes.js

```diff
@@ -41,7 +41,7 @@ export default function addBraveRoutes(r) {
     r.BRAVE_WALLET_NETWORKS = r.BRAVE_WALLET.createChild('/wallet/networks');
   }
 
-  r.BRAVE_HELP_TIPS = r.BASIC.createSection('/braveHelpTips', 'braveHelpTips')
```

Comment:
So does this move the help tips into the advanced section?

Reply:
```
 r.BRAVE_HELP_TIPS = r.ADVANCED.createSection('/braveHelpTips', 'braveHelpTips')
 ```
yes

Reply:
it is already located inside the advanced, but created in wrong section.

Reply:
👍